201-web-app-with-app-gateway-v2/azuredeploy.json

```diff
@@ -0,0 +1,285 @@
+        "appGwRoutingRuleName": "appGatewayRoutingRule"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2019-04-01",
+            "name": "[variables('virtualNetworkName')]",
+            "location": "[variables('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[variables('virtualNetworkAddressPrefix')]"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "[variables('virtualNetworkSubnetName')]",
+                        "properties": {
+                            "addressPrefix": "[variables('virtualNetworkSubnetPrefix')]",
+                            "serviceEndpoints":[
+                                {
+                                    "service": "Microsoft.Web",
+                                    "locations": [
+                                        "*"
+                                    ]
+                                }
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "name": "[variables('webAppPlanName')]",
+            "apiVersion": "2018-02-01",
+            "location": "[variables('location')]",
+            "tags": {},
+            "properties": {
+                 "reserved": "false"
+            },
+            "sku": {
+                "name": "[variables('webAppPlanSku')]",
+                "capacity": 1
+            }
+        },
+        {
+            "name": "[variables('webAppName')]",
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2018-11-01",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[variables('webAppPlanId')]",
+                "[variables('virtualNetworkId')]"
+            ],
+            "properties": {
+              "serverFarmId": "[variables('webAppPlanId')]",
+              "reserved": "false",
+              "siteConfig": {
+                  "http20Enabled": "true",
+                  "minTlsVersion": "1.2",
+                  "ipSecurityRestrictions": [
+                    {
+                    "vnetSubnetResourceId": "[variables('virtualNetworkSubnetId')]",
+                    "action": "Allow",
+                    "tag": "Default",
+                    "priority": 200,
+                    "name": "appGatewaySubnet",
+                    "description": "Isolate traffic to subnet containing Azure Application Gateway"
+                    }
+                ]
+              },
+              "httpsOnly": "false"
+            },
+            "identity": {
+                "type": "SystemAssigned"
```

Comment:
how is this used?

Reply:
It is not, but were just in the interest of being prepared. I have removed it.